start_bundle_coupled.R

```diff
@@ -121,6 +121,13 @@ if (length(argv) > 0) {
 
 if (! file.exists("output")) dir.create("output")
 
+# Check if dependencies for a REMIND model run are fulfilled
+if (packageVersion("lucode2") >= "0.31.0") {
+  lucode2::checkDeps(action = "stop")
+} else {
+  stop("REMIND requires lucode2 >= 0.31.0, please use a library snapshot containing it.")
+}
```

Comment:
and if you are friendly, say "use a library snapshot after 2022_10_21_R4" (this one is in the making)

Reply:
we now need lucode2 0.34.0, which would be the first snapshot containing it?

Reply:
2022_10_26_R4 will be the first snapshot containing it in some minutes.